bundle/next/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -1120,6 +1130,12 @@ spec:
                 - ingresses
               verbs:
                 - '*'
+            - apiGroups:
+                - networking.k8s.io
+              resources:
+                - ingresses
+              verbs:
+                - '*'
```

Comment:
This supercedes the permission above (line 473--482)

Reply:
The first one is is cluster roles another one just roles.